shared/openId4VCI/Utils.ts

```diff
@@ -111,6 +108,14 @@ export const getBody = async context => {
   };
 };
 
+export const getCredentialType = (context: any) => {
+  return context.selectedCredentialType?.credential_definition?.type
+    ? context.selectedCredentialType.credential_definition.type
+    : context.selectedIssuer?.credential_type
+    ? context.selectedIssuer.credential_type
+    : ['VerifiableCredential', 'MOSIPVerifiableCredential'];
```

Comment:
Extract this as const defaultCredentialType

Reply:
The Credential Type **['VerifiableCredential', 'MOSIPVerifiableCredential']**  is used nowhere else except this utility method. extracting it a const will introduce new variable which is unnecessary, so Its better inline. and the method name "getCredentialType" itself suggest its a credential type.